tests/plugins/wysiwygarea/manual/detachedwithplugins.html

```diff
@@ -0,0 +1,42 @@
+<button id="toggleButton">Toggle</button>
+<button id="setButton">Set data</button>
+
+<div id="editorContainer">
+	<div id="editor">
+		<p>Lorem ipsum dolor sit amet.</p>
+	</div>
+</div>
+
+<script>
+	embedTools.delayJsonp();
+
+	var editorElement = document.getElementById( 'editor' ),
+		editorContainer = document.getElementById( 'editorContainer' ),
+		editorContainerParent = editorContainer.parentNode,
+		editor = CKEDITOR.replace( editorElement, {
+			mathJaxLib: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-AMS_HTML',
+			embed_provider: '//ckeditor.iframe.ly/api/oembed?url={url}&callback={callback}',
+			extraPlugins: 'autoembed'
```

Comment:
This plugin is already included in bender tags, so no need to add it here as well.